src/wwwroot/app/services/loggerService.js

```diff
@@ -0,0 +1,36 @@
+const OFF = 0;
+const DEBUG_COEFFICIENT = 1;
+const LOG_COEFFICIENT = 2;
+const WARNING_COEFFICIENT = 3;
+const ERROR_COEFFICIENT = 4;
+var logLevel = DEBUG_COEFFICIENT;
+let _$log;
+
+export default class LoggerService {
+   constructor($log) {
+      'ngInject';
+      _$log = $log;
+   }
+
+   debug(...args) {
```

Comment:
When I talk about `...args` I mean that You will using it instead of `arguments` below